src/runtime/auto-prompt-manager.js

```diff
@@ -410,6 +420,29 @@ export class AutoPromptManager {
     return true;
   }
 
+  /**
+   * Determines what Audience Action prompt type should be shown.
+   *
+   * Show the first AutoPromptType passed in from Audience Actions.
+   * @param {string[]|undefined} actions
+   * @return {!AutoPromptType|undefined}
+   */
+  getAutoPromptType_(actions) {
```

Comment:
This would ensure the `actions.find()` method available
```suggestion
  getAutoPromptType_(actions = []) {
```

Reply:
Done.